release/schedule/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Release Schedule
+---
+
+WPE WebKit follows a 6-month development cycle, where:
+
+- There are two feature releases are done every year, typically in March and
+  September.
+- Within feature releases there may be any number of bug fix releases.
+- Development releases are the base for the next feature releases, and they
```

Comment:
"Development releases are the base for the feature releases that follow them. They do not follow..."